faas-runtime-to-kyma-migration/time-based-trigger/README.md

```diff
@@ -0,0 +1,120 @@
+
+```shell
+$ kubectl get functions   
+NAME              CONFIGURED   BUILT   RUNNING   RUNTIME    VERSION   AGE
+hello-timer       True         True    True      nodejs14   1         41s
+```
+
+At this point the Function is iddle. In the next step you will deploy a CronJob that will trigger it based on a desired schedule.
+
+### Create a Cronjob 
+
+#### Collect the trigger configuration:
+
+Export the Function name and namespace as environment variables.
+
+```shell
+export TRIGGER_FN_NAME={FUNCTION NAME}
+export TRIGGER_FN_NAMESPACE={FUNCTION NAMESPACE}
+```
+For example:
+
+```shell
+export TRIGGER_FN_NAME=hello-timer
+export TRIGGER_FN_NAMESPACE=default
+```
+
+Export the Function UID:
+```shell
+export TRIGGER_FN_UID=$(kubectl get function -n ${TRIGGER_FN_NAMESPACE} ${TRIGGER_FN_NAME} -o=jsonpath={".metadata.uid"})
+```
+
+All this data will allow to define the Function as the CronJob's owner so in case the Function is deleted from the cluster the CronJobs will be deleted as well.
+
+Export the schedule of the trigger in [cron format](https://en.wikipedia.org/wiki/Cron).
+```shell
+export TRIGGER_SCHEDULE={TRIGGER SCHEDULE}
+```
+For example `*/5 * * * *` to schedule execution every 5 minute.
+
+#### Create the CronJob
+
+Run the following kubectl command to create the CronJob:
+
+```shell
+cat <<EOF | kubectl create -f -
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  generateName: trigger-${TRIGGER_FN_NAME}-
+  namespace: ${TRIGGER_FN_NAMESPACE}
+  ownerReferences:
+    - apiVersion: serverless.kyma-project.io/v1alpha1
+      kind: Function
+      name: ${TRIGGER_FN_NAME}
+      uid: ${TRIGGER_FN_UID}
+spec:
+  schedule: "${TRIGGER_SCHEDULE}"
+  concurrencyPolicy: "Forbid"
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: cron-trigger
+              image: eu.gcr.io/kyma-project/external/curlimages/curl:7.70.0
+              imagePullPolicy: IfNotPresent
+              command:
+                - sh
+                - -c
+                - |
+                  curl ${TRIGGER_FN_NAME}.${TRIGGER_FN_NAMESPACE};
+                  x=\$?; curl -fsI -X POST http://localhost:15020/quitquitquit && exit \$x
+          restartPolicy: Never
+EOF
+
+```
+
+### Test the setup
+
+Inspect the Function logs. This can be done by a command similar to:
+
+```shell
+➜  ~ k logs hello-timer-mthpc-d6c879fb9-76cz8 function -f
+run at 2022-07-13T10:05:06.959Z
+run at 2022-07-13T10:10:09.077Z
+run at 2022-07-13T10:15:05.686Z
+```
+
+### Considerations
+
+ - This approach is not recommended if you need high accuracy in execution times. The kubernetes starts a CronJob (deploys a container ) and there is a variable few seconds delay before your Function gets executed. 
+ - You may consider running your logic in the CronJob container itself instead of having a CronJob calling your Function via it's internal address.
```

Comment:
```suggestion
 - You can consider running your logic in the Cron Job container, instead of having Cron Job calling your Function using its internal address.
```